settings/eztags.ini

```diff
@@ -25,6 +25,12 @@ TagCloudOverSolr=enabled
 # interface display when searching for tags in content edit
 MaxResults=24
 
+# Defines if auto suggest should use wild card search to match tags that
+# contains the string in the middle of the tag instead of checking if the tag
+# starts with the string
+# Default is disabled, so it will only match tags that starts with the string
+BroadMatchAutoComplete=disabled
```

Comment:
I'm not completely sold on "BroadMatch" notation, it's not really clear what this does from the setting name itself.

How about `AutoCompleteType` or something similar?